tests/test_inventory_client.py

```diff
@@ -1,4 +1,5 @@
 """Tests for the class ``InventoryClient``."""
+# pylint: disable=too-many-lines
```

Comment:
Do not exclude pylint rules on module level. Exclude it in the specific function. It's important to be explicit saying that **we know** we are disabling it for a specific case.

Reply:
Well the problem is the linter is complaining that the ``module`` has too many lines, so we need to disable it at the module level (at least it's the only way I know). TBH I agree with the linter that this file has too many lines, buy I think is something we could tackle in another PR.